README.md

```diff
@@ -137,18 +137,44 @@ Events:
 - Status
 - Workflow job
 
-# Development
+# Local development
 
-The bot requires some environment variables listed in
-[config.rs](./src/config.rs). They can, optionally, be set through an `.env`
-file which should be placed at the root of this repository. We provide an
-[example .env file](./.env.example) which can be used as a starting point.
+This project is a standard Rust project with some notable requirements.
+
+## Requiremnets
+
+Before you can generate a debug or release binary, we have some library
+requirements to install.
+
+```sh
+$ sudo apt install \
+    libssl-dev \
+    libclang-dev
+```
+
+## Environment variables
+
+This bot relies on the crate `dotenv`, which requires a `.env` file to
```

Comment:
It does not *require* the .env file as you can set the environment variables directly. That is why the original text said "optionally".

Also I see no reason to mention the `dotenv` crate here. That's an internal implementation detail which documentation readers should not have to care about.